vendor/github.com/Microsoft/hcsshim/internal/schema2/properties.go

```diff
@@ -44,4 +47,8 @@ type Properties struct {
 	SharedMemoryRegionInfo []SharedMemoryRegionInfo `json:"SharedMemoryRegionInfo,omitempty"`
 
 	GuestConnectionInfo *GuestConnectionInfo `json:"GuestConnectionInfo,omitempty"`
+
+	// Metrics is not part of the API for HCS but this is used for LCOW v2 to
+	// return the full cgroup metrics from the guest.
+	Metrics *v1.Metrics `json:"LCOWMetrics,omitempty"`
```

Comment:
@dcantah this unnecessarily drags in `github.com/containerd/cgroups/stats/v1` can we somehow avoid this?

Reply:
Definitely doable, need to look into it though.

Reply:
Ack @dcantah